spec/controllers/api/v1/authenticated_user.rb

```diff
@@ -0,0 +1,7 @@
+shared_context 'Authenticated User' do
```

Comment:
Althougth this is okay, i would not use shared context with a before because it will always call it

Reply:
Why not? I think it's okay